src/System.Private.Xml/tests/Xslt/XslTransformApi/CXslTransform.cs

```diff
@@ -523,13 +506,18 @@ public void XmlResolver7(InputType inputType, ReaderType readerType, TransformTy
             {
                 _output.WriteLine(e.ToString());
                 _output.WriteLine("Could not copy file to local. Some other issues prevented this test from running");
-                return; //return TEST_SKIPPED;
+                throw;
```

Comment:
As with earlier, it's not clear to me what value this try/catch is adding... seems like the catch block could be removed entirely, and the exception just allowed to propagate.